stdlib/2.7/ConfigParser.pyi

```diff
@@ -74,7 +74,7 @@ class RawConfigParser:
     def has_section(self, section: str) -> bool: ...
     def options(self, section: str) -> list[str]: ...
     def read(self, filenames: str) -> list[str]: ...
-    def readfp(self, fp: file, filename: str = ...) -> None: ...
+    def readfp(self, fp: BinaryIO, filename: str = ...) -> None: ...
```

Comment:
I think it's better to use `IO[str]` -- BinaryIO derives from that, but some other things return an IO[str] that's not a BinaryIO, and usually that works fine.